### ISSUE TITLE
能否提供可执行环境要求,总是报错

### ISSUE THREAD

**Issue by kkkparty** [shenweichen/GraphEmbedding#63]
![Uploading image.png…]()

**Comment by read2017** [comment]
This is my env that i spend couple hours to, finally it can work.
attention :            joblib  0.14.0

# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             4.5                       1_gnu  
absl-py                   1.0.0                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
ca-certificates           2021.10.26           h06a4308_2  
cachetools                5.0.0                    pypi_0    pypi
certifi                   2021.10.8        py38h06a4308_2  
charset-normalizer        2.0.11                   pypi_0    pypi
cycler                    0.11.0                   pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
flatbuffers               2.0                      pypi_0    pypi
fonttools                 4.29.1                   pypi_0    pypi
gast                      0.5.3                    pypi_0    pypi
gensim                    3.6.0                    pypi_0    pypi
google-auth               2.6.0                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.43.0                   pypi_0    pypi
h5py                      3.6.0                    pypi_0    pypi
idna                      3.3                      pypi_0    pypi
importlib-metadata        4.10.1                   pypi_0    pypi
joblib                    0.14.0                   pypi_0    pypi
keras                     2.8.0                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kiwisolver                1.3.2                    pypi_0    pypi
ld_impl_linux-64          2.35.1               h7274673_9  
libclang                  13.0.0                   pypi_0    pypi
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.3.0               h5101ec6_17  
libgomp                   9.3.0               h5101ec6_17  
libstdcxx-ng              9.3.0               hd4cf53a_17  
markdown                  3.3.6                    pypi_0    pypi
matplotlib                3.5.1                    pypi_0    pypi
ncurses                   6.3                  h7f8727e_2  
networkx                  2.1                      pypi_0    pypi
numpy                     1.22.2                   pypi_0    pypi
oauthlib                  3.2.0                    pypi_0    pypi
openssl                   1.1.1m               h7f8727e_0  
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 21.3                     pypi_0    pypi
pandas                    1.4.0                    pypi_0    pypi
pillow                    9.0.1                    pypi_0    pypi
pip                       21.2.4           py38h06a4308_0  
protobuf                  3.19.4                   pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pyparsing                 3.0.7                    pypi_0    pypi
python                    3.8.12               h12debd9_0  
python-dateutil           2.8.2                    pypi_0    pypi
pytz                      2021.3                   pypi_0    pypi
readline                  8.1.2                h7f8727e_1  
requests                  2.27.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.8                      pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scipy                     1.8.0                    pypi_0    pypi
setuptools                58.0.4           py38h06a4308_0  
six                       1.16.0                   pypi_0    pypi
smart-open                5.2.1                    pypi_0    pypi
sqlite                    3.37.2               hc218d9a_0  
tensorboard               2.8.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
tensorflow                2.8.0                    pypi_0    pypi
tensorflow-io-gcs-filesystem 0.24.0                   pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
tf-estimator-nightly      2.8.0.dev2021122109          pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tk                        8.6.11               h1ccaba5_0  
tqdm                      4.62.3                   pypi_0    pypi
typing-extensions         4.0.1                    pypi_0    pypi
tzdata                    2021e                hda174b7_0  
urllib3                   1.26.8                   pypi_0    pypi
werkzeug                  2.0.2                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0  
wrapt                     1.13.3                   pypi_0    pypi
xz                        5.2.5                h7b6447c_0  
zipp                      3.7.0                    pypi_0    pypi
zlib                      1.2.11               h7f8727e_4

**Comment by kkkparty** [comment]
thanks a lot for your help!



---原始邮件---
发件人: "blue ***@***.***&gt;
发送时间: 2022年2月8日(周二) 下午3:27
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [shenweichen/GraphEmbedding] 能否提供可执行环境要求,总是报错 (Issue #63)




 
This is my env that i spend couple hours to, finally it can work.
 attention :            joblib  0.14.0
 
Name                    Version                   Build  Channel
 
_libgcc_mutex             0.1                        main
 _openmp_mutex             4.5                       1_gnu
 absl-py                   1.0.0                    pypi_0    pypi
 astunparse                1.6.3                    pypi_0    pypi
 ca-certificates           2021.10.26           h06a4308_2
 cachetools                5.0.0                    pypi_0    pypi
 certifi                   2021.10.8        py38h06a4308_2
 charset-normalizer        2.0.11                   pypi_0    pypi
 cycler                    0.11.0                   pypi_0    pypi
 decorator                 5.1.1                    pypi_0    pypi
 flatbuffers               2.0                      pypi_0    pypi
 fonttools                 4.29.1                   pypi_0    pypi
 gast                      0.5.3                    pypi_0    pypi
 gensim                    3.6.0                    pypi_0    pypi
 google-auth               2.6.0                    pypi_0    pypi
 google-auth-oauthlib      0.4.6                    pypi_0    pypi
 google-pasta              0.2.0                    pypi_0    pypi
 grpcio                    1.43.0                   pypi_0    pypi
 h5py                      3.6.0                    pypi_0    pypi
 idna                      3.3                      pypi_0    pypi
 importlib-metadata        4.10.1                   pypi_0    pypi
 joblib                    0.14.0                   pypi_0    pypi
 keras                     2.8.0                    pypi_0    pypi
 keras-preprocessing       1.1.2                    pypi_0    pypi
 kiwisolver                1.3.2                    pypi_0    pypi
 ld_impl_linux-64          2.35.1               h7274673_9
 libclang                  13.0.0                   pypi_0    pypi
 libffi                    3.3                  he6710b0_2
 libgcc-ng                 9.3.0               h5101ec6_17
 libgomp                   9.3.0               h5101ec6_17
 libstdcxx-ng              9.3.0               hd4cf53a_17
 markdown                  3.3.6                    pypi_0    pypi
 matplotlib                3.5.1                    pypi_0    pypi
 ncurses                   6.3                  h7f8727e_2
 networkx                  2.1                      pypi_0    pypi
 numpy                     1.22.2                   pypi_0    pypi
 oauthlib                  3.2.0                    pypi_0    pypi
 openssl                   1.1.1m               h7f8727e_0
 opt-einsum                3.3.0                    pypi_0    pypi
 packaging                 21.3                     pypi_0    pypi
 pandas                    1.4.0                    pypi_0    pypi
 pillow                    9.0.1                    pypi_0    pypi
 pip                       21.2.4           py38h06a4308_0
 protobuf                  3.19.4                   pypi_0    pypi
 pyasn1                    0.4.8                    pypi_0    pypi
 pyasn1-modules            0.2.8                    pypi_0    pypi
 pyparsing                 3.0.7                    pypi_0    pypi
 python                    3.8.12               h12debd9_0
 python-dateutil           2.8.2                    pypi_0    pypi
 pytz                      2021.3                   pypi_0    pypi
 readline                  8.1.2                h7f8727e_1
 requests                  2.27.1                   pypi_0    pypi
 requests-oauthlib         1.3.1                    pypi_0    pypi
 rsa                       4.8                      pypi_0    pypi
 scikit-learn              1.0.2                    pypi_0    pypi
 scipy                     1.8.0                    pypi_0    pypi
 setuptools                58.0.4           py38h06a4308_0
 six                       1.16.0                   pypi_0    pypi
 smart-open                5.2.1                    pypi_0    pypi
 sqlite                    3.37.2               hc218d9a_0
 tensorboard               2.8.0                    pypi_0    pypi
 tensorboard-data-server   0.6.1                    pypi_0    pypi
 tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
 tensorflow                2.8.0                    pypi_0    pypi
 tensorflow-io-gcs-filesystem 0.24.0                   pypi_0    pypi
 termcolor                 1.1.0                    pypi_0    pypi
 tf-estimator-nightly      2.8.0.dev2021122109          pypi_0    pypi
 threadpoolctl             3.1.0                    pypi_0    pypi
 tk                        8.6.11               h1ccaba5_0
 tqdm                      4.62.3                   pypi_0    pypi
 typing-extensions         4.0.1                    pypi_0    pypi
 tzdata                    2021e                hda174b7_0
 urllib3                   1.26.8                   pypi_0    pypi
 werkzeug                  2.0.2                    pypi_0    pypi
 wheel                     0.37.1             pyhd3eb1b0_0
 wrapt                     1.13.3                   pypi_0    pypi
 xz                        5.2.5                h7b6447c_0
 zipp                      3.7.0                    pypi_0    pypi
 zlib                      1.2.11               h7f8727e_4
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
